server/admins/utils.py

```diff
@@ -144,6 +144,36 @@ def delete_music_data(request, **kwargs) -> response.JsonResponse:
         )
 
 
+def delete_comments(request, **kwargs) -> response.JsonResponse:
+    """Deletes exact comment specified by its ID
+
+    Args:
+        request
+        **kwargs
+
+    Returns:
+        response.JsonResponse
+    """
+    try:
+        comment = request.query_params.get("comment")
+        Comments.delete_data(comment=comment)
+
+        return response.JsonResponse(
+            {"success_status": True},
+            status=status.HTTP_200_OK,
+        )
+
+    except Exception as e:
+        return response.JsonResponse(
+            {
+                "error_status": True,
+                "success_status": False,
+                "exception": e
```

Comment:
Revealing the exception data can be a security threat - Better to not disclose exception data